src/pages/Order/OrderStyle.js

```diff
@@ -0,0 +1,184 @@
+import styled from 'styled-components';
+
+export const mixins = {
+  flexBox: (align = 'center', justify = '') => `
+    display: flex;
+    align-items: ${align};
+    justify-content: ${justify};
+  `,
+};
+
+export const OrderWrap = styled.div`
+  font-weight: 700;
+  ${mixins.flexBox('flex-start', 'space-between')}
+`;
+
+export const TotalOrderInfo = styled.div`
+  width: 730px;
+`;
+
+export const Header = styled.h1`
+  margin: 0px 0px 20px;
+  line-height: 36px;
+  font-size: 40px;
+  font-weight: 700;
+  color: rgb(0, 0, 0);
+`;
+
+export const Title = styled.div`
+  height: 56px;
+  margin-top: 20px;
+  border-bottom: 1px solid #afd485;
+  line-height: 56px;
+  font-size: 25px;
+  font-weight: bold;
+`;
+
+export const OrderInfo = styled.div`
+  overflow: hidden;
+  min-height: 150px;
+  margin: -3px;
+  padding: 23px 3px 3px;
+  font-size: 20px;
+`;
+
+export const ContentP = styled.p`
+  margin: ${({ margin }) => (margin ? '15px 0' : '')};
+  font-size: 20px;
+  font-family: ${({ font }) => font};
+`;
+
+export const ContentSpan = styled.span`
+  margin: 0px 0px 0px 6px;
+  line-height: 33px;
+  font-size: 15px;
+`;
+
+export const Checkbox = styled.input`
+  width: 20px;
+  height: 20px;
+  accent-color: #568a35;
+`;
+
+export const InputSpan = styled.div`
+  ${mixins.flexBox('flex-start')}
+  width: 72px;
+  max-height: 40px;
+  white-space: pre-line;
+  line-height: 18px;
+  font-size: 15px;
+  color: rgb(117, 117, 117);
+`;
+
+export const Input = styled.input`
+  display: inline-block;
+  width: 89%;
+  margin: 0;
+  margin-bottom: ${({ marginBottom }) => (marginBottom ? '15px' : '')};
+  padding: 8px 15px 9px;
+  border: 1px solid #dbdbdb;
+  border-radius: 4px;
+  color: #000;
+  line-height: 21px;
+  font-size: 15px;
+  background-color: #fff;
+  &:focus {
+    box-shadow: 0 0 5px #568a35;
+    border-color: #568a35;
+    outline: none;
+  }
+`;
+
+export const BoxWrap = styled.div`
+  ${mixins.flexBox('center')}
+  padding: ${({ padding }) => (padding ? '' : '8px 0px')};
```

Comment:
```suggestion
  padding: ${({ padding }) => (padding || '8px 0px')};
```
객체에서 특정 key에 접근했는데 정의되지않은 key에 접근하면 undefined로 평가됩니다.
undefined는 falsy값으로, 암묵적으로 false로 평가됩니다.

* padding의 값이 있으면 속성값에 빈 스트링을 부여하고 
* padding이 없을때 속성값이 8px 0px로 평가된다는 표현식입니다.

* `||` 논리합 연산자는 단축평가의 기능도 있어서,  좌항이 false로 평가되면 우항의 값을 반환하는 기능이 있습니다.

* `padding || "8px 0px"` 이 뜻은
padding이 false로 평가 되는 값이면 속성 값이 '8px 0px'로 수렴되는 식입니다.

팀원들에게도 복음을 전파해주세요

Reply:
널리 퍼뜨리겠습니다.!